dashboard/android/app/src/main/AndroidManifest.xml

```diff
@@ -40,6 +40,7 @@
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
         </activity>
+         <receiver android:name= "com.dexterous.flutterlocalnotifications.ScheduledNotificationBootReceiver" android:exported="true"/>
```

Comment:
Can you explain why you add this receiver ?

Reply:
because the app didn't work without it 
"there is probelem in package FLUTTER LOCAL NOTIFICATIONS"